playground/AzureContainerApps/AzureContainerApps.AppHost/Program.cs

```diff
@@ -3,8 +3,17 @@
 
 #pragma warning disable ASPIREACADOMAINS001 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
 
+using Azure.Provisioning.AppContainers;
+
 var builder = DistributedApplication.CreateBuilder(args);
 
+builder.AddContainerAppEnvironment("infra")
```

Comment:
Are you limited to one container app environment per app host or is it possible to have multiple?

Reply:
The plan is to support multiple

Reply:
In this PR to simplify things and make progress, I've decided to throw if there's more than one.